### PR TITLE
RF AT_09.02_005| My Views>Add description

### PR DESCRIPTION
**Pull request by KatyaKaterina33**
https://trello.com/c/uRnD0Yad/1183-rf-myviewsadddescription-at-0902005-my-views-add-description

### CODE REVIEW

File: cypress/e2e/myViewsAddDescriptions.cy.js [hunk 1]
```diff
@@ -10,13 +11,16 @@ describe('My Views add description', () => {
         cy.get('#description div:nth-child(1)').should('have.text','DESCRIPTION')
     })
 
-    let inputText = 'description';
     it('AT 09.02.005| My Views> Add description', function () {
         cy.contains('My Views').click();
```
Comment (DariaMas7) [inline cypress/e2e/myViewsAddDescriptions.cy.js:15]: 'My Views should be moved to fixtures

File: cypress/e2e/myViewsAddDescriptions.cy.js [hunk 2]
```diff
@@ -10,13 +11,16 @@ describe('My Views add description', () => {
...
         cy.get('#description-link').click();
-        cy.get('.jenkins-input').should('be.visible').clear().type(inputText);
+        cy.get('.jenkins-input')
+            .should('be.visible')
+            .clear()
```
Comment (DariaMas7) [inline cypress/e2e/myViewsAddDescriptions.cy.js:19]: Why are you adding   .clear() 
Your TC states that you are adding a description not editing it, I would simple delete it

File: cypress/e2e/myViewsAddDescriptions.cy.js (hunk 1 above)
Comment (KatyaKaterina33) [reply cypress/e2e/myViewsAddDescriptions.cy.js:15]: done

File: cypress/e2e/myViewsAddDescriptions.cy.js (hunk 2 above)
Comment (KatyaKaterina33) [reply cypress/e2e/myViewsAddDescriptions.cy.js:19]: At some test runs, the description field contains some text, so before entering new descriptions, I clear this field. I changed the test case, added a line: "Make sure the input description field does not have any description."

Comment (DariaMas7) [reply cypress/e2e/myViewsAddDescriptions.cy.js:20]: Oh, I see. I don't know why it runs in this way. Thank you for clarification

**Review by DariaMas7 — APPROVED** [review]

**Review by InnaBorovenska — APPROVED** [review]

**Pull request merged**